app/controllers/citizens/own_homes_controller.rb

```diff
@@ -2,12 +2,12 @@ module Citizens
   class OwnHomesController < BaseController
     def show
       @legal_aid_application = LegalAidApplication.find(session[:current_application_ref])
-      @form = Citizens::OwnHomeForm.new(current_params)
+      @form = LegalAidApplications::OwnHomeForm.new(model: legal_aid_application)
     end
 
     def update
       @legal_aid_application = LegalAidApplication.find(session[:current_application_ref])
```

Comment:
Same here, this line can be removed as it has been replaced by legal_aid_application method